packages/Spinner/Spinner.jsx

```diff
@@ -43,6 +43,20 @@ const OpaqueContainer = styled.div({
   opacity: 0.06,
 })
 
+const recursiveMap = (children, fn) =>
```

Comment:
This could work, but I'd like to have this tested with 20 child nodes first. 

- Create a form with 20 core-input components, and two core-heading components
- Toggle and untoggle the spinner with a button, see if there is noticeable jank - test with dev tools running at 6x CPU slowdown
- Also set `inert={true}` to each child to provide a progressive enhancement for touch devices